### ISSUE TITLE
xcode-test@2.2.0 (exit code 1) 

### ISSUE THREAD

**Issue by tedbrosby** [bitrise-io/steps-xcode-test#110]
My build URL: https://app.bitrise.io/build/0fc3672f3c64f79c  
The error message is:  
`failed to get simulator udid, error: %!(EXTRA *errors.errorString=failed to get latest os version, error: failed to find latest os version for (iOS) - (iPhone 6))`  
Why are default options failing?  
I've wasted 10+ builds trying to get the default build to work...  
I'm using Mac OS 10.14, Xcode 10.2.x stack with xcode-test 2.2.0.  
I have 0 issues running tests from local machine. It seems to be an environment issue on the Mac OS VM's end.

**Comment by BirmacherAkos** [comment]
Hi!

Thanks for the issue report.
We know about this issue. It seems the new `simctl` binary which was shipped with the  Xcode 10.2 beta doesn't lists the installed simulators on the VM.
We are using the `simctl` binary in our steps to get the list of the available simulators.

On Xcode 10.1 stack it looks like this:
```
== Devices ==
-- iOS 10.0 --
    iPhone 5 (1BC1F4E1-65CD-4648-BC25-750E7D33EF12) (Shutdown) 
    iPhone 5s (0E934528-0E08-4FF2-853D-A0BCBF1CAA29) (Shutdown) 
    iPhone 6 (CC448165-4768-4B96-98C0-6FC3041D4A53) (Shutdown) 
    iPhone 6 Plus (4238B76A-A54A-4D04-9F6A-298F4443DBDC) (Shutdown) 
    iPhone 6s (469F9E30-153E-4FFB-A70B-92FDBA097C0C) (Shutdown) 
    iPhone 6s Plus (4BA2E087-1B37-46D3-B754-4CBD5089EE06) (Shutdown) 
    iPhone SE (BC6B39CE-58AD-4D84-8E77-F94B90184C17) (Shutdown) 
    iPad Air (23340507-43F7-406E-812B-4E47B064F243) (Shutdown) 
    iPad Air 2 (0F6EDACB-23CC-4666-A780-B55E52F1AF69) (Shutdown) 
    iPad Pro (9.7 inch) (3C684880-EF42-4134-AE03-78F47D569295) (Shutdown) 
    iPad Pro (12.9 inch) (3E5C40A8-1FA1-45E5-98D9-6AD6BD91187E) (Shutdown) 
-- iOS 10.1 --
    iPhone 5 (E1E32F08-3E25-4648-8353-564D4F8014C2) (Shutdown) 
    iPhone 5s (6E1A0D5F-02E0-441C-9418-30C8E1C3F32D) (Shutdown) 
    iPhone 6 (C396CE68-0C87-47D8-B10D-FF119E0C034B) (Shutdown) 
    iPhone 6 Plus (75A1B902-206D-468C-B99E-9D90FC8023B2) (Shutdown) 
    iPhone 6s (595912F6-0AA7-4799-BDED-35A1B8CC5F0B) (Shutdown) 
    iPhone 6s Plus (BAB3A0D9-8F9C-4742-9C4F-F5A577F263C4) (Shutdown) 
    iPhone 7 (49836CC1-B19C-406E-B858-7F77C40BBBFC) (Shutdown) 
    iPhone 7 Plus (BB02A764-468B-491E-B525-7CA345D3BF69) (Shutdown) 
    iPhone SE (E069F116-8898-49A5-B077-2395069F3373) (Shutdown) 
    iPad Air (0C638804-F725-44BA-993B-EF7FD49B69CE) (Shutdown) 
    iPad Air 2 (04EC405F-2867-4610-87C3-37415FA7487D) (Shutdown) 
    iPad Pro (9.7 inch) (214232F4-740C-4646-A879-A5B637550B0D) (Shutdown) 
    iPad Pro (12.9 inch) (7752D0D3-19D1-4FF4-AB74-B742C9E76275) (Shutdown) 
-- iOS 10.2 --
    iPhone 5 (1B4A0352-3515-4336-8CA5-94E36C5A5483) (Shutdown) 
    iPhone 5s (B311B167-3F10-4269-9F39-B6EBB6D9DABE) (Shutdown) 
    iPhone 6 (9C53A131-5541-4449-BF54-0C0D09D03383) (Shutdown) 
    iPhone 6 Plus (52FE718A-471B-4D0D-8977-D533575674E6) (Shutdown) 
    iPhone 6s (5AC0CFDA-8EBC-436C-8B4E-EF39B9CBA338) (Shutdown) 
    iPhone 6s Plus (9A66AB38-E83B-4D66-A544-1E0A8476C4F8) (Shutdown) 
    iPhone 7 (0138E6BA-367A-43FB-82AC-C4B6DB4A34B8) (Shutdown) 
    iPhone 7 Plus (808C41AE-2827-41AE-A5FE-11F8CCE92C28) (Shutdown) 
    iPhone SE (06C19DF5-AA4A-4599-B863-E0DD42E85EC7) (Shutdown) 
    iPad Air (1712526A-8733-4848-9824-99A115EAAB90) (Shutdown) 
    iPad Air 2 (8AA559F1-50B7-4DE9-8F57-4BF9CCC3DF8A) (Shutdown) 
    iPad Pro (9.7 inch) (BFA5C588-0221-4E28-B68F-B97E32CB031E) (Shutdown) 
    iPad Pro (12.9 inch) (763E44B9-16AB-4100-A64C-E30355C3680F) (Shutdown) 
-- iOS 10.3 --
    iPhone 5 (936719EF-5D55-4690-ADE1-0992CDA08E8C) (Shutdown) 
    iPhone 5s (2DAE33C0-1DD4-45BF-B753-A30407753DCF) (Shutdown) 
    iPhone 6 (7406DF26-C665-49F9-A70D-386154BB2E5A) (Shutdown) 
    iPhone 6 Plus (339AE3AB-B086-43CD-93CC-AFF83C461961) (Shutdown) 
    iPhone 6s (28E1D3D9-FDE2-4140-B96E-AF57D1EE122D) (Shutdown) 
    iPhone 6s Plus (0575DB7D-ED2E-4DC9-A1B6-3E3808AE1ECF) (Shutdown) 
    iPhone 7 (A0BA2184-D920-4969-80B7-046F6AB858FD) (Shutdown) 
    iPhone 7 Plus (2AD55D08-2D2A-44F8-9103-6813C8074FBF) (Shutdown) 
    iPhone SE (38DC0115-09DC-4759-B3C8-E0DDBEAB9D79) (Shutdown) 
    iPad Air (ADC28AD4-424C-44F6-A5D2-B33F423F63C7) (Shutdown) 
    iPad Air 2 (6263BCAA-B848-4BC1-A9E7-9C0181CCDFC6) (Shutdown) 
    iPad (5th generation) (0F8FE10A-440A-4989-A4EB-ACCEE71CD011) (Shutdown) 
    iPad Pro (9.7 inch) (520F6793-93A3-41B7-8CEE-64465B2A3BDF) (Shutdown) 
    iPad Pro (12.9 inch) (68FBC6B8-8169-48E2-8DED-30D7F374163F) (Shutdown) 
    iPad Pro (12.9-inch) (2nd generation) (F461D734-2E14-446C-B30E-69E26B268216) (Shutdown) 
    iPad Pro (10.5-inch) (B15A0003-2202-4F49-BEFC-CA688C4102EE) (Shutdown) 
-- iOS 11.0 --
    iPhone 5s (7DA98698-FA00-4204-9ADA-E543A6928C32) (Shutdown) 
    iPhone 6 (3433330F-0596-417B-AB05-28605095E618) (Shutdown) 
    iPhone 6 Plus (1DF5FA4C-6CAE-4F7B-B067-BF70E9463778) (Shutdown) 
    iPhone 6s (01E12D2D-0ADD-4ED9-B965-2234B105DD4A) (Shutdown) 
    iPhone 6s Plus (FAA9E584-1350-4568-8A52-9A8823A1C19D) (Shutdown) 
    iPhone 7 (412FE9DA-4EB7-49D0-BB86-739481050230) (Shutdown) 
    iPhone 7 Plus (37AFC10D-3E3E-48B3-BFBD-E80CBA1DF8F0) (Shutdown) 
    iPhone 8 (7B0BCBE3-B543-4E83-A0B9-356137113548) (Shutdown) 
    iPhone 8 Plus (460C62F8-64D3-44B8-9D73-5850FED925C5) (Shutdown) 
    iPhone SE (58C8A896-4A63-45D0-9B67-411C4B12C5E2) (Shutdown) 
    iPhone X (20BE6EBA-F1AA-4B32-892E-B1BA624ED4A0) (Shutdown) 
    iPad Air (B1B84DC8-0C07-4D51-9C74-144BB53E97DD) (Shutdown) 
    iPad Air 2 (3B4EB828-50C2-4CFC-A968-C55B53F7AA13) (Shutdown) 
    iPad (5th generation) (94583701-9FB5-484A-AE50-419BF9B242FE) (Shutdown) 
    iPad Pro (9.7-inch) (7371F2FC-92EC-425B-BEB4-1AEC0DCDFAFD) (Shutdown) 
    iPad Pro (12.9-inch) (469C0581-B131-41AE-AE73-DC357889FF3A) (Shutdown) 
    iPad Pro (12.9-inch) (2nd generation) (87A02007-A029-4937-8B5C-2F0C934A363A) (Shutdown) 
    iPad Pro (10.5-inch) (7B8372A6-0E60-45E5-86C8-659D09D96FFC) (Shutdown) 
-- iOS 11.1 --
    iPhone 5s (9F5EB2AF-DC0E-4E6C-817A-DAA2382553AC) (Shutdown) 
    iPhone 6 (38F04F31-1C46-40C3-A294-F8329490E4AA) (Shutdown) 
    iPhone 6 Plus (2692B50E-1455-44D2-AA0F-3414BDE41494) (Shutdown) 
    iPhone 6s (0568222A-9DB9-4CE6-BCBB-F08A40F8D5C5) (Shutdown) 
    iPhone 6s Plus (4AAB3968-4B82-4AE7-BE30-BCA432492089) (Shutdown) 
    iPhone 7 (C914A1BB-8560-4F26-8E05-C3DDE854D6FC) (Shutdown) 
    iPhone 7 Plus (9ACD7FD2-B629-445E-B7F8-5535743A48DC) (Shutdown) 
    iPhone 8 (3216930A-CE76-457E-A0D0-71BB112AD091) (Shutdown) 
    iPhone 8 Plus (CB8A1DF6-48C4-4F0D-9CF5-FF8B800D2B02) (Shutdown) 
    iPhone SE (CAFB36E8-9EA4-4792-859C-B57BF355A3D9) (Shutdown) 
    iPhone X (4FF3978E-9BAD-40DE-9F88-FEB74B841044) (Shutdown) 
    iPad Air (F59D2248-BA9D-471E-ACDE-6C29324B7CF5) (Shutdown) 
    iPad Air 2 (0B7B18C1-54E3-4B37-A5F4-A62E66858246) (Shutdown) 
    iPad (5th generation) (BFF0803E-B253-42AA-8B07-235A9FC7C8D0) (Shutdown) 
    iPad Pro (9.7-inch) (174AD485-B85F-44ED-A4A7-AE87B1EA3A85) (Shutdown) 
    iPad Pro (12.9-inch) (3E2F3AC2-5342-4A3A-A77A-F07E7491DBD2) (Shutdown) 
    iPad Pro (12.9-inch) (2nd generation) (6A56B4B4-6D8F-4799-AF6D-0E9B60F29CD6) (Shutdown) 
    iPad Pro (10.5-inch) (FF300239-3437-4330-8FCB-4CDF69187DC9) (Shutdown) 
-- iOS 11.2 --
    iPhone 5s (FD70A493-725E-40F4-8537-CA4A2FF5BF0D) (Shutdown) 
    iPhone 6 (68619613-3AF5-4DB4-9A43-F2F051EA23EA) (Shutdown) 
    iPhone 6 Plus (976585D0-FE34-4481-BC31-D96F266AE905) (Shutdown) 
    iPhone 6s (C2685413-BEA1-43E7-9B52-81347E095E52) (Shutdown) 
    iPhone 6s Plus (6EDF976C-BE6F-4E07-96B7-F0E045381B73) (Shutdown) 
    iPhone 7 (D97EE991-0D93-4C1D-952E-B1CF4F6DF0BC) (Shutdown) 
    iPhone 7 Plus (0100970B-17F8-4140-82B0-FEC1891835D3) (Shutdown) 
    iPhone 8 (E1E9AD4A-1003-4BFE-B0B7-A8A671F1749A) (Shutdown) 
    iPhone 8 Plus (3C5D61A1-6A8E-431D-909A-F29B75B24351) (Shutdown) 
    iPhone SE (7BDD444F-1D7F-435F-8704-B952CFAAE03E) (Shutdown) 
    iPhone X (596259FC-A36F-4F8E-8CC5-DA7D9C0C720F) (Shutdown) 
    iPad Air (79B1DCD1-2194-439E-83B8-EF5E113C5143) (Shutdown) 
    iPad Air 2 (33758FFB-84CC-475B-AB7C-32540585F5D8) (Shutdown) 
    iPad (5th generation) (A10C2C9A-E7A3-4BC8-B6EC-16C45E69E735) (Shutdown) 
    iPad Pro (9.7-inch) (6C70EAA3-AD19-4DBE-B111-9E49DDD4578C) (Shutdown) 
    iPad Pro (12.9-inch) (5D187671-62FC-458E-B830-7F98A86BF697) (Shutdown) 
    iPad Pro (12.9-inch) (2nd generation) (62517CB2-53EC-46AE-92D3-F22C03895FD1) (Shutdown) 
    iPad Pro (10.5-inch) (1C06D01D-3519-4CA4-9D11-4EE6E8F8AF89) (Shutdown) 
-- iOS 11.3 --
    iPhone 5s (3B5F6E29-C50A-4F89-84BF-ACDFFA541215) (Shutdown) 
    iPhone 6 (51B4DDC9-2525-4EA2-82D3-BB2B2F120FF7) (Shutdown) 
    iPhone 6 Plus (A9000D96-325B-4A55-BFD0-3B94BCB1BB8D) (Shutdown) 
    iPhone 6s (95F1111A-5407-4F4A-8284-F421F02E9B36) (Shutdown) 
    iPhone 6s Plus (CB021A76-FF05-4328-ADFE-E65767C7BF3E) (Shutdown) 
    iPhone 7 (F3D53653-E8D2-4002-8820-CE98973B60A1) (Shutdown) 
    iPhone 7 Plus (1F2305CA-C172-413E-87D5-90C28675BBB4) (Shutdown) 
    iPhone 8 (17F81D58-8345-4C34-8AD4-682C0A0B4F9B) (Shutdown) 
    iPhone 8 Plus (913CEF3F-CD40-4154-A751-9475F4B10DB1) (Shutdown) 
    iPhone SE (94E4F413-5373-4D96-9BFD-D97A55799434) (Shutdown) 
    iPhone X (1927B628-AF9E-4E3B-9160-58479F29F801) (Shutdown) 
    iPad Air (D711ED93-BA2F-466E-ABAD-A58DD9CEA03C) (Shutdown) 
    iPad Air 2 (1E8A4830-DAFD-4D06-885D-4AEFD35A93EF) (Shutdown) 
    iPad (5th generation) (F3327EC1-525B-4261-B3E1-FFB944F38515) (Shutdown) 
    iPad Pro (9.7-inch) (0ADB6B90-5480-4DA5-9478-8E81A0BA340A) (Shutdown) 
    iPad Pro (12.9-inch) (D1059A9A-74E8-44F7-BC01-541384E297EF) (Shutdown) 
    iPad Pro (12.9-inch) (2nd generation) (D46753A7-B58A-4224-A179-D84DDE49F63E) (Shutdown) 
    iPad Pro (10.5-inch) (38DA0E72-5A58-41B4-8767-3B5A70125BDB) (Shutdown) 
-- iOS 11.4 --
    iPhone 5s (DCE9E9CE-F882-419A-8E6F-4D507F070D00) (Shutdown) 
    iPhone 6 (B121EEB0-BBAE-4347-A486-C965C7333C7E) (Shutdown) 
    iPhone 6 Plus (01CD87E9-AFC9-4D2A-8802-0887A839AC31) (Shutdown) 
    iPhone 6s (39415903-4025-441D-9D7E-16A4318AE24B) (Shutdown) 
    iPhone 6s Plus (99845B80-4338-44FD-954A-FE02EB43748A) (Shutdown) 
    iPhone 7 (112B64E7-0157-46BD-AEA9-9B8B92E14498) (Shutdown) 
    iPhone 7 Plus (FE505FF3-402B-4A3C-B03D-903DF580F98D) (Shutdown) 
    iPhone 8 (8AD0D10C-C530-4142-B3EF-7374F13F685E) (Shutdown) 
    iPhone 8 Plus (8E205E2E-2FBE-4F83-A5CE-D9ADC585DBC8) (Shutdown) 
    iPhone SE (7170581D-390D-4CAA-B7D5-D6E18F0522FE) (Shutdown) 
    iPhone X (649D6DEA-AD22-4B3D-A433-1BE08EB87F42) (Shutdown) 
    iPad Air (B5C8012B-F8A0-4B23-A9EA-69BCDEB6C6DA) (Shutdown) 
    iPad Air 2 (1DA34860-3883-4AC4-854B-1EF1527A56DE) (Shutdown) 
    iPad (5th generation) (392CB9C3-D6FD-447D-B65B-3A78AA558DDD) (Shutdown) 
    iPad Pro (9.7-inch) (88987A07-CC24-40D6-9D7D-9019AC60ABB0) (Shutdown) 
    iPad Pro (12.9-inch) (B530BD8F-5E89-4911-A479-2A3AF4B90F66) (Shutdown) 
    iPad Pro (12.9-inch) (2nd generation) (5E0B0B18-7626-4464-B084-764FCA4FC13B) (Shutdown) 
    iPad Pro (10.5-inch) (F842ABD1-3DC2-4635-946B-6266FB361FA8) (Shutdown) 
-- iOS 12.0 --
    iPhone 5s (BC0478D8-7C56-4901-BEB6-4BC3FD3F962A) (Shutdown) 
    iPhone 6 (17FF9CD2-5A26-4A04-8C39-8ED23E1A5352) (Shutdown) 
    iPhone 6 Plus (D3CBC161-DA9B-4D48-AC4E-DEB36CF4DF91) (Shutdown) 
    iPhone 6s (373FC75A-0A68-4E94-B2B9-9447120762A0) (Shutdown) 
    iPhone 6s Plus (981048BF-C72C-4D7D-B906-F94BFAB4CE37) (Shutdown) 
    iPhone 7 (6887715C-9987-46B1-A620-29CE9ECB063C) (Shutdown) 
    iPhone 7 Plus (31B2280A-A98E-4803-A12B-EF68E2E0CFBB) (Shutdown) 
    iPhone 8 (009E42CF-E13A-48F6-90D0-25184423F307) (Shutdown) 
    iPhone 8 Plus (18980FB1-7896-490E-BF82-16782A24E976) (Shutdown) 
    iPhone SE (10B4FFB3-E288-4B2B-A3B4-AA30FD4C9333) (Shutdown) 
    iPhone X (E687D748-3E77-45C6-902A-AA0FE54C33DF) (Shutdown) 
    iPhone XS (1FD108B7-0850-4B93-BFFF-317592EA89CA) (Shutdown) 
    iPhone XS Max (DCE32AF7-D46B-4313-84A1-BAAB738E0C9C) (Shutdown) 
    iPhone XR (97639580-B179-4EA9-8D1A-ECD4925D4A1B) (Shutdown) 
    iPad Air (5FBD3F85-8986-46F8-A21C-8C731109C190) (Shutdown) 
    iPad Air 2 (E668B230-CB1E-4C42-9C8A-5DBF3D48B5EA) (Shutdown) 
    iPad (5th generation) (99F68FD8-F327-4F97-9911-D46095E0122E) (Shutdown) 
    iPad Pro (9.7-inch) (19E30A21-52DC-4BFA-AD4F-E6C1CB977BC3) (Shutdown) 
    iPad Pro (12.9-inch) (8C45EBC7-E40E-4C1C-BDEE-FBD9A80A657E) (Shutdown) 
    iPad Pro (12.9-inch) (2nd generation) (4661C2D0-0955-4D6C-8400-AF192FB1D23A) (Shutdown) 
    iPad Pro (10.5-inch) (A51BD1FF-A881-4480-AE17-4869E3BDD4FE) (Shutdown) 
    iPad (6th generation) (5A08C58F-1C88-44EF-BB72-88226499BBE3) (Shutdown) 
-- iOS 12.1 --
    iPhone 5s (8FAA11AE-D765-4106-B6D3-028C277F2720) (Shutdown) 
    iPhone 6 (C2BE722D-DD62-470E-9766-46080B3939DC) (Shutdown) 
    iPhone 6 Plus (AA2759AC-626E-498E-A99B-64C64227D8AC) (Shutdown) 
    iPhone 6s (4DCB7AA0-3990-4560-9A88-604253C4F820) (Shutdown) 
    iPhone 6s Plus (B4953501-E313-4A8E-8012-452CEA2B4D01) (Shutdown) 
    iPhone 7 (313A440B-FE02-4018-BA1F-0E808D149DAD) (Shutdown) 
    iPhone 7 Plus (6F044FFD-6E95-4E56-B7E9-EEAF2FFFD40E) (Shutdown) 
    iPhone 8 (8A1F9274-CD16-485A-A84C-6FCD603E2FAF) (Shutdown) 
    iPhone 8 Plus (F7913C29-86C2-4BB0-BA91-BA90CABBCE10) (Shutdown) 
    iPhone SE (FBF3D849-8970-4F88-8F73-5D80C787B18B) (Shutdown) 
    iPhone X (3CE004E0-042F-48DB-A7D3-F4528357CC42) (Shutdown) 
    iPhone XS (13DBF587-0422-4034-A017-709C91D123D5) (Shutdown) 
    iPhone XS Max (CE95F4B5-4CA8-429A-BCA7-23C63C04B099) (Shutdown) 
    iPhone XR (8C079DE7-8611-4988-B39A-21640C43DB0A) (Shutdown) 
    iPad Air (6E86682E-30C3-40C3-B154-92458DEC0BFF) (Shutdown) 
    iPad Air 2 (8A8B1DAC-82F3-4FCB-BC87-1CBD486102BB) (Shutdown) 
    iPad (5th generation) (FC415C74-6E8B-414C-B9FB-DDD330F78E24) (Shutdown) 
    iPad Pro (9.7-inch) (9CCA62DF-1825-4BDE-A6D3-8E9E1CB18AF4) (Shutdown) 
    iPad Pro (12.9-inch) (122B9277-2ACC-4D77-89F7-2194CF497E09) (Shutdown) 
    iPad Pro (12.9-inch) (2nd generation) (05C1EA25-B858-4C8A-A2F7-D9D729FFDEC7) (Shutdown) 
    iPad Pro (10.5-inch) (51BF6423-DF5D-4141-9C1D-2CE2B8982D9F) (Shutdown) 
    iPad (6th generation) (7FC23596-0903-4BB9-887B-0DDEBE587D0C) (Shutdown) 
    iPad Pro (11-inch) (1603277A-53D5-4A7A-B5B0-441B65700241) (Shutdown) 
    iPad Pro (12.9-inch) (3rd generation) (25A096EE-7CF2-41E8-9EE6-BF700D110613) (Shutdown) 
-- tvOS 10.0 --
    Apple TV 1080p (EEAD31A0-9FFD-4D33-B728-E6B33A5A7981) (Shutdown) 
-- tvOS 10.1 --
    Apple TV 1080p (59B44E04-A433-4B47-91C7-5C83F16A9B02) (Shutdown) 
-- tvOS 10.2 --
    Apple TV 1080p (B438ADA4-1B7F-4677-B6D4-CC91DF4706BF) (Shutdown) 
-- tvOS 11.0 --
    Apple TV (83700EF7-E4B2-4DB4-97DC-71A2DE2EA33E) (Shutdown) 
    Apple TV 4K (9512C535-83E4-408E-BD51-3A74FBAB9EFE) (Shutdown) 
    Apple TV 4K (at 1080p) (E1E830F1-A0C9-43D6-92B4-1F8CA072BE8B) (Shutdown) 
-- tvOS 11.1 --
    Apple TV (D889EB5A-5AFB-4284-8EAE-85E129AC7D95) (Shutdown) 
    Apple TV 4K (B54D15BF-C532-4A44-97C5-2D212B3017D1) (Shutdown) 
    Apple TV 4K (at 1080p) (1202D836-A810-466B-9083-E3D4CE108EB5) (Shutdown) 
-- tvOS 11.2 --
    Apple TV (1D8E9E81-62A2-4F5C-AB2B-6AFA5301C914) (Shutdown) 
    Apple TV 4K (EA5202DB-EE01-4575-91C9-32236CF61002) (Shutdown) 
    Apple TV 4K (at 1080p) (6BB456FF-8B92-415A-9DC7-81CE5EE70F0F) (Shutdown) 
-- tvOS 11.3 --
    Apple TV (321EE4C9-5FAE-46BE-9B72-2B9C1E2C75DD) (Shutdown) 
    Apple TV 4K (75EBAA3F-AF90-497E-B469-49E3E2B0F3D7) (Shutdown) 
    Apple TV 4K (at 1080p) (8292EF5E-D953-4B80-8E93-BED434C436DC) (Shutdown) 
-- tvOS 11.4 --
    Apple TV (CF81B0D4-290C-4754-94F1-B0E7E721364B) (Shutdown) 
    Apple TV 4K (98C4E9CE-AB9E-4D7A-B88C-40C1568E211D) (Shutdown) 
    Apple TV 4K (at 1080p) (C8F6D1BF-BCF2-4E8F-BB21-C219864B0A5D) (Shutdown) 
-- tvOS 12.0 --
    Apple TV (8D83AA8A-5CC3-4146-875C-73C6EDE66813) (Shutdown) 
    Apple TV 4K (577C322C-2225-471A-B0AE-29B2C22ED37D) (Shutdown) 
    Apple TV 4K (at 1080p) (D666D747-75E5-4710-B73C-5BC9E23BB76D) (Shutdown) 
-- tvOS 12.1 --
    Apple TV (9C330632-507B-44E7-885F-670375801846) (Shutdown) 
    Apple TV 4K (7D06F5F9-B2DB-484E-9871-B14ADCDE2FC4) (Shutdown) 
    Apple TV 4K (at 1080p) (F8DC3A99-EFBE-48B1-A0D6-3B8C9DA13E96) (Shutdown) 
-- watchOS 3.1 --
    Apple Watch - 38mm (CCF0D58A-5359-41D9-8345-19AE44262F65) (Shutdown) 
    Apple Watch - 42mm (5140137F-01CD-447F-AE58-920A6BBE5088) (Shutdown) 
    Apple Watch Series 2 - 38mm (01A71605-0CFC-4749-A2A5-A1AE74B784B8) (Shutdown) 
    Apple Watch Series 2 - 42mm (5A7BBB18-CAF3-4573-9063-2F7A2DA94189) (Shutdown) 
-- watchOS 3.2 --
    Apple Watch - 38mm (ABB91D81-1B76-476E-BF9D-4226DA58A119) (Shutdown) 
    Apple Watch - 42mm (D68C7B5A-54D0-448E-94EC-795F55D9A07A) (Shutdown) 
    Apple Watch Series 2 - 38mm (2EC5E246-0150-4F76-B706-CE22F454CBFE) (Shutdown) 
    Apple Watch Series 2 - 42mm (B1FF5137-B484-400B-80AC-90E69AA01B17) (Shutdown) 
-- watchOS 4.0 --
    Apple Watch - 38mm (D5FF2075-DE7E-4028-84DF-6EF313177070) (Shutdown) 
    Apple Watch - 42mm (4887A39F-98ED-4305-B292-01789D826B40) (Shutdown) 
    Apple Watch Series 2 - 38mm (BB9A6364-52DA-49CF-9137-6B39BED3AB02) (Shutdown) 
    Apple Watch Series 2 - 42mm (1B573065-B0ED-4FD7-A1DB-B2DC05E8D6F7) (Shutdown) 
    Apple Watch Series 3 - 38mm (23AAFAD6-EC4C-4831-B390-C470C657580C) (Shutdown) 
    Apple Watch Series 3 - 42mm (119351ED-F00D-4231-8553-37DDB085CBD2) (Shutdown) 
-- watchOS 4.1 --
    Apple Watch - 38mm (DFEF0622-C307-4DC3-B993-1B5ADA8DCF8D) (Shutdown) 
    Apple Watch - 42mm (86C9D06F-F384-44AB-8863-EE6ABECAC098) (Shutdown) 
    Apple Watch Series 2 - 38mm (2D5A97B8-9C5F-471C-8725-980BB7D259B1) (Shutdown) 
    Apple Watch Series 2 - 42mm (42260F46-197F-4846-AF8E-F2E2F5AAE57D) (Shutdown) 
    Apple Watch Series 3 - 38mm (60D4DF12-C344-457A-BA91-7AFFB63570C9) (Shutdown) 
    Apple Watch Series 3 - 42mm (462757FF-C767-4D11-A87F-2B5A30F06E1E) (Shutdown) 
-- watchOS 4.2 --
    Apple Watch - 38mm (62774FC3-561A-47F5-9657-09786A95A9EA) (Shutdown) 
    Apple Watch - 42mm (B9574713-CF04-43AB-BE70-25AFFE3978F3) (Shutdown) 
    Apple Watch Series 2 - 38mm (A1DC747C-3647-4C8A-915C-3323E1A02408) (Shutdown) 
    Apple Watch Series 2 - 42mm (C6A5086B-B58C-46DC-84EE-6091096A300B) (Shutdown) 
    Apple Watch Series 3 - 38mm (A6194950-1586-4850-9FC0-D3795CCA6920) (Shutdown) 
    Apple Watch Series 3 - 42mm (9DE815ED-0CA6-41FE-9987-1A509A257422) (Shutdown) 
-- watchOS 5.0 --
    Apple Watch Series 2 - 38mm (5BD4E430-EA31-4F16-9B7B-113A12F678D4) (Shutdown) 
    Apple Watch Series 2 - 42mm (3F92464C-F8F7-4C76-9578-D088267B8A75) (Shutdown) 
    Apple Watch Series 3 - 38mm (03B200AA-5524-4D1D-B36E-43AE7BBD6560) (Shutdown) 
    Apple Watch Series 3 - 42mm (A0F8DFB0-BE88-4B77-9E11-A8F10D372E14) (Shutdown) 
    Apple Watch Series 4 - 40mm (15868360-40D9-4AEA-8A0C-88F75EBBBA46) (Shutdown) 
    Apple Watch Series 4 - 44mm (3FE1057F-DFDB-4B54-8C21-58B7A1FEA20D) (Shutdown) 
-- watchOS 5.1 --
    Apple Watch Series 2 - 38mm (BE4CA38D-8D18-4C24-A577-F04D76800E48) (Shutdown) 
    Apple Watch Series 2 - 42mm (52FE3CDB-38EA-4682-BA7D-03B94362D987) (Shutdown) 
    Apple Watch Series 3 - 38mm (E302CA08-BD1A-43EC-9F48-A453577CE7D8) (Shutdown) 
    Apple Watch Series 3 - 42mm (6C6E1702-163C-4EC5-8FE2-8C58040AA422) (Shutdown) 
    Apple Watch Series 4 - 40mm (A24E4AB3-E16B-4D7B-8A11-7909F62C1E68) (Shutdown) 
    Apple Watch Series 4 - 44mm (FEFCFA66-F99B-432F-A146-CB0385D936C4) (Shutdown)
```

On Xcode 10.2  it looks like this:
```
== Devices ==
-- iOS 12.2 --
    iPhone 5s (0B5280BC-409B-40B3-99CC-16428125084E) (Shutdown)
-- tvOS 12.2 --
-- watchOS 5.2 --
    Apple Watch Series 2 - 38mm (7939DBD6-F3C2-4A00-ADF0-BE8AD927CB76) (Shutdown)
    Apple Watch Series 2 - 42mm (347F2B09-A9B6-4AB2-8F05-AAB6FF2F4C11) (Shutdown)
    Apple Watch Series 3 - 38mm (EC95B52F-49F6-4248-B460-F3125D09E3EF) (Shutdown)
    Apple Watch Series 3 - 42mm (233AA45F-E1A7-4C05-9826-E3CDD2D4E160) (Shutdown)
```

---

 The simulators are already downloaded to the VM, but the simctl is not listing them.

---


The Tooling Team is currently working on the solution.

**Comment by tedbrosby** [comment]
Thanks for the answer😊.  
In the end I switched to a Fast lane unit test lane.  
It's smooth after that but I'm glad for the list of simulators.

**Comment by andrewtheis** [comment]
@birmacher Can we use the above ID list as a workaround?

**Comment by BirmacherAkos** [comment]
@andrewtheis 
Well, you can use the `iPhone 5s` simulator:
https://discuss.bitrise.io/t/xcode-test-2-2-0-fails-to-find-simulator/8529/5?u=birmacherakos

In this week we will release the stable Xcode 10.2 version. Hopefully, it will fix it.

**Comment by BirmacherAkos** [comment]
Hi!

We have released the Xcode 10.2 final version
https://discuss.bitrise.io/t/xcode-10-2-final-arrived/8577

And it's `simctl` has all the simulators again :tada:
So all the Xcode steps with simulators should work now.

I'm closing this issue, but feel free to re-open it if the issue still persists for you.
Happy Building 🚀
